README.md

```diff
@@ -1 +1,78 @@
-# sumologic-kubernetes-collection
\ No newline at end of file
+# sumologic-kubernetes-collection
+
+# Installation
+
+First use `kops` to deploy your Kubernetes cluster in your AWS account.
+
+## Setting up Fluentd
+
+1. Clone this repo and navigate to `./deploy/kubernetes`.
+2. Create an HTTP source in your Sumo Logic account to get an HTTP Source URL.
+3. Run `echo -n '<YOUR_HTTP_URL>' | base64` to encode your HTTP Source URL.
+4. In `secret-sumologic.yaml`, under `endpoint-metrics:`, replace `XXXX` with the encoded URL from step 3.
+5. Ensure you are in the correct directory `./deploy/kubernetes`. Install the fluentd plugins using `kubectl`:
+```
+kubectl apply -f ./namespace-sumologic.yaml -f ./secret-sumologic.yaml -f ./deployment-fluentd.yaml
+```
+****Note:** Ideally we'd be able to run `kubectl apply -f .`, which applies all files in the directory. However there is a longstanding [issue](https://github.com/kubernetes/kubernetes/issues/16448) with `kubectl` that causes it to be unable to find the created namespace before trying to create the objects specified in `deployment-fluentd.yaml` which depend on that namespace. Thus for now we have to specify the order of the files (or rename the files such that they are alphabetically in the correct order).*
+
+## Setting up Prometheus
+
+1. Install `helm`.
+```
+brew install kubernetes-helm
+```
+2. Create the necessary RBAC role in your Kubernetes cluster by creating a [`tiller-rbac.yaml`](https://docs.google.com/document/d/1Iu1zqTusPALc0I7rfIz9S7yLQ941JVfAO2qZWmovKwE/edit?usp=sharing) file. Then run
+```
```

Comment:
Could we add a tiller-rbac file to the repo? Rather than link to a Google doc?

Reply:
yes. see https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/11